scripts/archive_collection.py

```diff
@@ -17,83 +17,76 @@
 $ python scripts/archive_collection.py \
```

Comment:
this should be renamed `upload_to_zenodo.py`

Reply:
yea, agree

Reply:
fixed